web/themes/webspark/renovation/src/components/accordions/accordion.twig

```diff
@@ -1,14 +1,14 @@
 {% set is_icon = content.field_icon|render %}
 
 {% set link_id = paragraph.id() %}
-{# Card #}
-<div class="card card-foldable mt-3{% if color != 'default' %} {{ color }}{% endif %}">
+{# Accordion #}
+<div class="accordion-item mt-3{% if color != 'default' %} {{ color }}{% endif %}">
 
-  {# Card Header #}
-  <div class="card-header{% if is_icon  %} card-header-icon{% endif %}">
-    <h3>
+  {# Accordion Header #}
+  <div class="accordion-header{% if is_icon  %} accordion-header-icon{% endif %}">
+    <h4>
```

Comment:
Question - was the change to h4 from h3 driven by the templates? We recently had some changes to WS2 and UDS related to the proper header level to use to avoid skipping over h3s and I want to make sure this isn't a regression. I believe @bebaps worked on that, so his memory may be better here.

Reply:
Hi @mlsamuelson 
I reverted this change, the thing was h3 was not working properly after new class changes(as it works in unity). But, it was a SCSS issue. 
So, Accordion is using h3 again and working properly now.
Commit : https://github.com/ASUWebPlatforms/webspark-ci/pull/100/commits/7121690330f4a982a9d17f208a9ca6747a796d63